packages/file_selector/file_selector_ios/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: file_selector_ios
 description: iOS implementation of the file_selector plugin.
 repository: https://github.com/flutter/plugins/tree/main/packages/file_selector/file_selector_ios
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+file_selector%22
-version: 0.5.0
+version: 0.6.0
```

Comment:
A readme update should just be a bugfix version change, which for versions <1.0 is the build number by Dart package convention. So this should be 0.5.0+1.

Reply:
That makes sense, initially I was going to update this to 0.9.0 since all other endorsed packages were skipped to that version number when endorsed, but the tooling wouldn't allow that.

Reply:
The desktop ones changed not because they were endorsed, but because they had crazy-low versions when they were part of FDE that made them sound super-experimental, which wasn't actually the case. It doesn't matter if they match, matching just seemed like a reasonable arbitrary version to pick since they were as old as the app-facing plugin.

Reply:
Good to know, thank you.